src/modules/fancyzones/FancyZonesLib/trace.cpp

```diff
@@ -244,6 +253,32 @@ void Trace::FancyZones::QuickLayoutSwitched(bool shortcutUsed) noexcept
         TraceLoggingBoolean(shortcutUsed, QuickLayoutSwitchedWithShortcutUsed));
 }
 
+void Trace::FancyZones::MoveNewWindowIntoZone(IZoneSet* activeSet) noexcept
+{
+    auto const zoneInfo = GetZoneSetInfo(activeSet);
+    TraceLoggingWrite(
+        g_hProvider,
+        EventMoveNewWindowIntoZone,
+        ProjectTelemetryPrivacyDataTag(ProjectTelemetryTag_ProductAndServicePerformance),
+        TraceLoggingKeyword(PROJECT_KEYWORD_MEASURE),
+        TraceLoggingValue(reinterpret_cast<void*>(activeSet), ActiveSetKey),
```

Comment:
How would this value be displayed in telemetry? We should make sure all values in telemetry events are human-readable. Anyway, it looks like useless value to log here.

Reply:
Now that I check it, it doesn't show much information indeed:
```
"data": {
"ActiveSet": 2058591956464,
"NumberOfZones": 4,
"NumberOfWindows": 1
}
```

The reason for including it is that it's included on the other telemetry events as well.

Reply:
Might this be to have an understading if the user is moving between different or the same active set? It seems to act as an ID for the current session.

Reply:
@dedavis6797 
Any idea why do we need `ActiveSet` here? Take into account that it was present for old telemetry events as well.